PoGo.NecroBot.CLI/Config/Translations/translation.hu.json

```diff
@@ -202,15 +202,15 @@
     },
     {
       "Key": "accountNotVerified",
-      "Value": "A fiók nincs megereősítve! Érdekes..."
+      "Value": "A fiók nincs megereősítve! "
     },
     {
       "Key": "openingGoogleDevicePage",
       "Value": "A Google eszköz oldalának megnyitása. Másold be a kódot CTRL+V"
     },
     {
       "Key": "couldntCopyToClipboard",
-      "Value": "Nem lehet másolni a vágólapra, csináld manuálisan"
+      "Value": "Nem lehet másolni a vágólapra, gépeld be"
```

Comment:
gépeld be means type it in, csináld manuálisan means do it manually but it should be csináld kézzel.